alicloud/service_alicloud_alikafka.go

```diff
@@ -171,6 +173,111 @@ func (alikafkaService *AlikafkaService) DescribeAlikafkaTopic(id string) (*alika
+	alikafkaSaslUser := &alikafka.SaslUserVO{}
+
+	parts, err := ParseResourceId(id, 2)
+	if err != nil {
+		return alikafkaSaslUser, WrapError(err)
+	}
+	instanceId := parts[0]
+	username := parts[1]
+
+	request := alikafka.CreateDescribeSaslUsersRequest()
+	request.InstanceId = instanceId
+	request.RegionId = alikafkaService.client.RegionId
+
+	wait := incrementalWait(3*time.Second, 5*time.Second)
+	var raw interface{}
+
+	err = resource.Retry(5*time.Minute, func() *resource.RetryError {
+		raw, err = alikafkaService.client.WithAlikafkaClient(func(alikafkaClient *alikafka.Client) (interface{}, error) {
+			return alikafkaClient.DescribeSaslUsers(request)
+		})
+		if err != nil {
+			if IsExceptedError(err, AlikafkaThrottlingUser) {
+				wait()
+				return resource.RetryableError(err)
+			}
+			return resource.NonRetryableError(err)
+		}
+		addDebug(request.GetActionName(), raw, request.RpcRequest, request)
+		return nil
+	})
+
+	if err != nil {
+		return alikafkaSaslUser, WrapErrorf(err, DefaultErrorMsg, id, request.GetActionName(), AlibabaCloudSdkGoERROR)
+	}
+
+	userListResp, _ := raw.(*alikafka.DescribeSaslUsersResponse)
+	addDebug(request.GetActionName(), raw, request.RpcRequest, request)
+
+	for _, v := range userListResp.SaslUserList.SaslUserVO {
+		if v.Username == username {
+			return &v, nil
+		}
+	}
+	return alikafkaSaslUser, WrapErrorf(Error(GetNotFoundMessage("AlikafkaSaslUser", id)), NotFoundMsg, ProviderERROR)
+}
+
+func (alikafkaService *AlikafkaService) DescribeAlikafkaSaslAcl(id string) (*alikafka.KafkaAclVO, error) {
+	alikafkaSaslAcl := &alikafka.KafkaAclVO{}
+
+	parts, err := ParseResourceId(id, 6)
+	if err != nil {
+		return alikafkaSaslAcl, WrapError(err)
+	}
+	instanceId := parts[0]
+	username := parts[1]
+	aclResourceType := parts[2]
+	aclResourceName := parts[3]
+	aclResourcePatternType := parts[4]
+	aclOperationType := parts[5]
+
+	request := alikafka.CreateDescribeAclsRequest()
+	request.InstanceId = instanceId
+	request.RegionId = alikafkaService.client.RegionId
+	request.Username = username
+	request.AclResourceType = aclResourceType
+	request.AclResourceName = aclResourceName
+
+	wait := incrementalWait(3*time.Second, 5*time.Second)
+	var raw interface{}
+
+	err = resource.Retry(5*time.Minute, func() *resource.RetryError {
+		raw, err = alikafkaService.client.WithAlikafkaClient(func(alikafkaClient *alikafka.Client) (interface{}, error) {
+			return alikafkaClient.DescribeAcls(request)
+		})
+		if err != nil {
+			if IsExceptedError(err, AlikafkaThrottlingUser) {
+				wait()
+				return resource.RetryableError(err)
+			}
+			return resource.NonRetryableError(err)
+		}
+		addDebug(request.GetActionName(), raw, request.RpcRequest, request)
+		return nil
+	})
+
+	if err != nil {
+		if e, ok := err.(*errors.ServerError); ok &&
+			(e.ErrorCode() == AlikafkaSubscriptionNotFound || e.ErrorCode() == AlikafkaTopicNotFound) {
```

Comment:
I think the line 263-264 can be instead by `IsExceptedErrors(AlikafkaSubscriptionNotFound, AlikafkaTopicNotFound)`